packages/appium/docs/scripts/build-reference.js

```diff
@@ -0,0 +1,34 @@
+const {fs} = require('@appium/support');
+const {log, LANGS, DOCS_DIR} = require('./utils');
+const path = require('path');
+const monorepoRoot = path.resolve(__dirname, '..', '..', '..', '..');
+const typedocOut = path.resolve(monorepoRoot, 'typedoc-docs');
+const {exec} = require('teen_process');
+
+const typedocMap = [[['commands', 'appium_base_driver.md'], ['base_driver.md']]];
+
+async function main() {
+  log.info('Generating typedoc reference material');
+  await exec('npm', ['run', 'typedoc'], {cwd: monorepoRoot});
+
+  for (const lang of LANGS) {
+    const langRefPath = path.resolve(DOCS_DIR, lang, 'reference');
+    await fs.rimraf(langRefPath);
+    await fs.mkdirp(langRefPath);
+    for (const [from, to] of typedocMap) {
+      const sourcePath = path.resolve(typedocOut, ...from);
+      const destPath = path.resolve(langRefPath, ...to);
+      log.info(`Copying ${sourcePath} to ${destPath}`);
+      await fs.copyFile(sourcePath, destPath);
```

Comment:
I assume doing the copy in parallel would be much faster
Also, do we only have files there and not folders? Otherwise we probably need something like https://github.com/appium/appium-ios-simulator/blob/5725bf7c352112e132c227ccd7839e4b2fd56db5/test/utils.js#L4

Reply:
we're copying a small number of specific files, currently just one :-)